lib/site_prism/element_container.rb

```diff
@@ -158,25 +144,23 @@ def create_nonexistence_checker(element_name, *find_args)
     def create_waiter(element_name, *find_args)
       method_name = "wait_for_#{element_name}"
       create_helper_method(method_name, *find_args) do
-        define_method(method_name) do |timeout = Capybara.default_max_wait_time, *runtime_args|
-          result = Capybara.using_wait_time(timeout) do
+        define_method(method_name) do |timeout = nil, *runtime_args|
```

Comment:
`timeout = nil` doesn't need setting here. Can be set on 149

Reply:
@luke-hill Since this is defining methods the user calls It needs to be defaulted to something, otherwise a user would have to specify a timeout every time they call it

Reply:
Probably a lack of my knowledge for now. Will ignore this comment.